pallets/creditcoin/src/migrations/v7.rs

```diff
@@ -20,6 +22,23 @@ pub(crate) fn migrate<T: Config>() -> Weight {
 	crate::weights::WeightInfo::<T>::migration_v7(n)
 }
 
+#[cfg(feature = "try-runtime")]
+#[allow(dead_code)]
+pub(crate) fn pre_upgrade<T: Config>() -> Result<(), &'static str> {
+	Ok(())
+}
+
+#[cfg(feature = "try-runtime")]
+#[allow(dead_code)]
```

Comment:
Interestingly enough Clippy doesn't complain about the same functions in `v6.rs`. 

@nathanwhit any idea why that is?

Reply:
FTR https://github.com/gluwa/creditcoin/blob/dev/pallets/creditcoin/src/migrations/mod.rs#L13-L28

I will look more into it.